packages/devtools_app/lib/src/vm_service_wrapper.dart

```diff
@@ -766,6 +766,36 @@ class VmServiceWrapper implements VmService {
     ).isSupported(supportedVersion: supportedVersion);
   }
 
+  /// Retrieves the full string value of a [stringRef].
+  ///
+  /// The string value stored with the [stringRef] is returned unless the value
+  /// is truncated in which an extra getObject call is issued to return the
+  /// value. If the [stringRef] has expired so the full string is unavailable,
+  /// [onUnavailable] is called to return how the truncated value should be
+  /// displayed. If [onUnavailable] is not specified, an exception is thrown
+  /// if the full value cannot be retrieved.
+  Future<String> retrieveFullStringValue(
+    String isolateId,
+    InstanceRef stringRef, {
+    String onUnavailable(String truncatedValue),
+  }) async {
+    if (stringRef == null) return null;
+    if (stringRef.valueAsStringIsTruncated != true)
```

Comment:
you can just write `!stringRef.valueAsStringIsTruncated`

Reply:
valueAsStringIsTruncated  looks like it may be null. would be nice to have null safety so this didn't matter.
expect when we get null safety support there will be a lint for this case.
for now I'd rather just keep this consistent with what this method looked like back before I moved it to a central location.

Reply:
sg